### ISSUE TITLE
Unable to access the argocd-server svc

### ISSUE THREAD

**Issue by sbin64** [argoproj/argo-cd#21240]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

After installing argocd using helm chart, not able to access the argocd-server through ingress on AKS cluster. 

Is there any bug with helm chart installation?

**To Reproduce**

Install argocd in a different namespace instead of argocd ns by default
Try to access the service through ingress as mentioned here https://argo-cd.readthedocs.io/en/latest/operator-manual/ingress/#kubernetesingress-nginx

**Screenshots**

![img_20241216_185657](https://github.com/user-attachments/assets/1880852c-25e8-4acb-a08b-464867e0ad76)
![img_20241216_185724](https://github.com/user-attachments/assets/cd69a224-d823-4c55-bd18-8fa47c2c0de1)
![img_20241217_164234](https://github.com/user-attachments/assets/e348f712-642f-4f07-a9f2-d923ecd448f9)
![img_20241217_155458](https://github.com/user-attachments/assets/593369c3-4f1e-482d-bfcf-bedcf8b94dc2)

**Version**

```shell

Paste the output from `argocd version` here.
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by andrii-korotkov-verkada** [comment]
What's your ArgoCD version? Do you have any useful logs?

**Comment by saiben10** [comment]
Argocd version - 4.5.10 I am using

**Comment by andrii-korotkov-verkada** [comment]
Is it a Helm chart version? Can you run `argocd version`, please?

**Comment by sbin64** [comment]
@andrii-korotkov-verkada Sai hadn't installed argocd cli, instead he installed argocd using helm chart. you can see the deployments are healthy from screenshot. Is it needed to install cli alongside?

**Comment by saiben10** [comment]
Yes I installed only Argocd using helm chart, I didn't installed Argocd cli

**Comment by andrii-korotkov-verkada** [comment]
No, you don't need a cli. That's just one of the ways to get server version info. Can you find it (e.g. 2.12.x, 2.13.x) or let me know the command you've used to install the helm chart, please?

**Comment by saiben10** [comment]
<img width="305" alt="image" src="https://github.com/user-attachments/assets/ac7a9d98-8eef-4651-afb2-94f2326bf95b" />

**Comment by sbin64** [comment]
Thanks @saiben10 your argocd version is 2.3.3

Could you add your helm install command to install argocd?
e.g. `helm install argocd ....`

**Comment by sbin64** [comment]
Is this like installing from a marketplace in the UI?

**Comment by saiben10** [comment]
yes even though i tried using helm commands also same issue i am facing

**Comment by sbin64** [comment]
can you update your version to v2.13.2?

**Comment by saiben10** [comment]
degrade the version?

**Comment by sbin64** [comment]
> degrade the version?

upgrade it to 2.13.2

BTW what is your updated or current version?

**Comment by saiben10** [comment]
version is 2.3.3

**Comment by sbin64** [comment]
> version is 2.3.3

plz update and try.

**Comment by saiben10** [comment]
okay sure

**Comment by sbin64** [comment]
@saiben10 any update? If you want, I can catch up with you in the zoom call.

**Comment by sbin64** [comment]
@saiben10 let us know. else we will have to close this issue.

**Comment by saiben10** [comment]
okay,
Thanks
